parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -227,7 +227,9 @@ if $FULL_INSTALL_REQUIRED; then
 fi
 
 {{- /* re-enable unattended upgrades */}}
+{{- if EnableUnattendedUpgrade }} 
 rm -f /etc/apt/apt.conf.d/99periodic
```

Comment:
how about we remove this, and then in cloud init, we set the correct value (enabled/disabled) based on the bool?